chpl/chpl-service/src/main/java/gov/healthit/chpl/manager/impl/CertifiedProductManagerImpl.java

```diff
@@ -28,9 +28,9 @@
 import gov.healthit.chpl.dao.CQMResultDAO;

```

Comment:
This class has an update method and a createFromPending method. Should those also trigger all caches being cleared? They can result in new certified products (would change search results) or developer status changes.